src/azure.js

```diff
@@ -1,8 +1,61 @@
+var _           = require('lodash');
 var azure       = require('fast-azure-storage');
 var api         = require('./v1');
 
 api.declare({
   method:     'get',
+  method:     'get',
+  route:      '/azure/:account/table/:table/read-write',
+  route:      '/azure/accounts',
+  name:       'azureAccounts',
+  input:      undefined,
+  output:     'azure-account-list-response.json#',
+  stability:  'stable',
+  scopes:     [['auth:azure-table-access:list-accounts']],
+  title:      "List Accounts Managed by Auth",
+  description: [
+    "Retrieve a list of all Azure accounts managed by Taskcluster Auth.",
+  ].join('\n')
+}, function(req, res) {
+  return res.reply({accounts: _.keys(this.azureAccounts)});
+});
+
+api.declare({
+  method:     'get',
+  route:      '/azure/:account/tables',
+  name:       'azureAccountTables',
+  query: {
+    continuationToken: /^[A-Za-z][A-Za-z0-9]{2,62}$/,
+  },
+  input:      undefined,
+  output:     'azure-table-list-response.json#',
+  stability:  'stable',
+  scopes:     [['auth:azure-table-access:<account>/list-tables']],
+  title:      "List Tables in an Account Managed by Auth",
+  description: [
+    "Retrieve a list of all tables in an account.",
+  ].join('\n')
+}, async function(req, res) {
+  let account = req.params.account;
+  let continuationToken  = req.query.continuationToken || null;
+
+  if (!req.satisfies({account})) { return; }
+
+  let table = new azure.Table({
+    accountId:  account,
+    accessKey:  this.azureAccounts[account]
+  });
+
+  let result = await table.queryTables({continuationToken});
+  if (result.nextTableName) {
+    result.continuationToken = result.nextTableName;
+  }
+  delete result.nextTableName;
+  return res.reply(result);
```

Comment:
prefer:
```js
let data = {tables: result.tables};
if (result.nextTableName) {
  data.continuationToken = result.nextTableName;
}
return res.reply(data);
```
Just as long, but you won't have an error when/if we add properties in `fast-azure-storage`.

IMO, we should augment `res.reply` so that it removes properties that has the value `undefined`. If we serialized to JSON they wouldn't be there either... The only reason we can just do: `res.reply({tables: result.tables, continuationToken: result.nextTableName || undefined});` is because `ajv` complains about `undefined`... Which is stupid because when serialized `{a: undefined}` becomes `"{}"`